### PR TITLE
Colored rules

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

- Add the ability to give horizontal, vertical and cross rules a color

> Why is it necessary?

- To make it easier to distinguish between different rules

Fixes #___

## Checklist

- [ ] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [x] Unit tests added or updated
- [ ] Documentation added or updated
- [ ] Example(s) added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [ ] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/CrossRule.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ class CrossRule extends mix(PixiTrack).with(RuleMixin, VerticalRuleMixin) {
   }
 
   drawHorizontalRule(graphics) {
-    let stroke = colorToHex('black');
+    let stroke = colorToHex(this.options.color ? this.options.color : 'black');
```
Comment (flekschas) [inline app/scripts/CrossRule.js:40]:
```suggestion
    let stroke = colorToHex(this.options.color || 'black');
```

Shorter

File: app/scripts/CrossRule.js [hunk 2]
```diff
@@ -86,22 +86,19 @@ class CrossRule extends mix(PixiTrack).with(RuleMixin, VerticalRuleMixin) {
...
+      `translate(${this.position[0]},${this.position[1]})`,
     );
 
     track.appendChild(output);
 
-    let stroke = 'black';
-    if (this.highlighted) {
-      stroke = 'red';
-    }
+    const stroke = this.options.color ? this.options.color : 'black';
```
Comment (flekschas) [inline app/scripts/CrossRule.js:94]:
```suggestion
    const stroke = this.options.color || 'black';
```

Shorter

File: app/scripts/HorizontalRule.js [hunk 3]
```diff
@@ -9,7 +9,9 @@ export const HorizontalRuleMixin = Mixin(
   superclass =>
     class extends superclass {
       drawHorizontalRule(graphics) {
-        let stroke = colorToHex('black');
+        let stroke = colorToHex(
+          this.options.color ? this.options.color : 'black',
```
Comment (flekschas) [inline app/scripts/HorizontalRule.js:13]:
```suggestion
          this.options.color || 'black',
```

Shorter

File: app/scripts/HorizontalRule.js [hunk 4]
```diff
@@ -99,15 +101,12 @@ class HorizontalRule extends mix(PixiTrack).with(
...
+      `translate(${this.position[0]},${this.position[1]})`,
     );
 
     track.appendChild(output);
 
-    let stroke = 'black';
-    if (this.highlighted) {
-      stroke = 'red';
-    }
+    const stroke = this.options.color ? this.options.color : 'black';
```
Comment (flekschas) [inline app/scripts/HorizontalRule.js:109]:
```suggestion
    const stroke = this.options.color || 'black';
```

Toastbrot 😅

File: app/scripts/SeriesListItems.js [hunk 5]
```diff
@@ -91,7 +91,7 @@ export const getSeriesItems = (
         <span styleName="context-menu-span">
           {x.options && x.options.name && x.options.name.length
             ? x.options.name
-            : x.uid}
+            : x.type}
```
Comment (flekschas) [inline app/scripts/SeriesListItems.js:94]: Just curious, does this fix a bug? Seems unrelated to the coloring.

File: app/scripts/VerticalRule.js [hunk 6]
```diff
@@ -9,7 +9,9 @@ export const VerticalRuleMixin = Mixin(
   superclass =>
     class extends superclass {
       drawVerticalRule(graphics) {
-        let stroke = colorToHex('black');
+        let stroke = colorToHex(
+          this.options.color ? this.options.color : 'black',
```
Comment (flekschas) [inline app/scripts/VerticalRule.js:13]:
```suggestion
          this.options.color || 'black',
```

File: app/scripts/VerticalRule.js [hunk 7]
```diff
@@ -85,15 +87,12 @@ export default class VerticalRule extends mix(PixiTrack).with(
...
+      `translate(${this.position[0]},${this.position[1]})`,
     );
 
     track.appendChild(output);
 
-    let stroke = 'black';
-    if (this.highlighted) {
-      stroke = 'red';
-    }
+    const stroke = this.options.color ? this.options.color : 'black';
```
Comment (flekschas) [inline app/scripts/VerticalRule.js:95]:
```suggestion
    const stroke = this.options.color || 'black';
```

File: app/scripts/SeriesListItems.js (hunk 5 above)
Comment (pkerpedjiev) [reply app/scripts/SeriesListItems.js:94]: Can't get anything past you ;-) Yeah, it fixes a small bug.
